README.rst

```diff
@@ -46,6 +46,7 @@ The chatbot service is deployed via Docker Compose.  You can set up a similar
 environment by running::
 
     docker compose build --pull
+    docker compose run prisma migrate deploy
     docker compose up -d
 
 
```

Comment:
We should probably add the Postgres variables to the configuration chapter of the readme.  I'm already adding the `CHAINLIT_AUTH_SECRET` documentation to the auth branch's changes.